ansible_collections/arista/avd/roles/eos_designs/python_modules/metadata/avdstructuredconfig.py

```diff
@@ -56,5 +56,6 @@ def metadata(self) -> dict | None:
             "platform": self.shared_utils.platform,
             "cv_tags": self._cv_tags(),
             "cv_pathfinder": self._cv_pathfinder(),
+            "system_mac_address": self.shared_utils.system_mac_address,
```

Comment:
Reordering since the tags can become huge and this one will be missed.
```suggestion
            "system_mac_address": self.shared_utils.system_mac_address,
            "cv_tags": self._cv_tags(),
            "cv_pathfinder": self._cv_pathfinder(),
```